libs/designsystem/testing-base/src/lib/mock-components.ts

```diff
@@ -3,7 +3,6 @@ import { MockAppComponent } from './components/mock.app.component';
 import { MockAvatarComponent } from './components/mock.avatar.component';
 import { MockBadgeComponent } from './components/mock.badge.component';
 import { MockButtonComponent } from './components/mock.button.component';
-import { MockCalendarComponent } from './components/mock.calendar.component';

```

Comment:
Why has the Calendar mock been removed? Please restore...

Reply:
I've restored the mock. However something strange happens, when I make a commit. When I make a commit, the system actually removes the mock from mock-compenent.ts itself, giving me a modified mock file as unstaged change.

Reply:
This error occured simply because I had incompatible exports. I used export * from instead of export { } from.